datafusion/sqllogictest/test_files/expr.slt

```diff
@@ -1100,24 +1100,23 @@ SELECT date_part('microsecond', timestamp '2020-09-08T12:00:12.12345678+00:00')
 query error DataFusion error: Internal error: unit Nanosecond not supported
 SELECT date_part('nanosecond', timestamp '2020-09-08T12:00:12.12345678+00:00')
 
-
 query I
 SELECT date_part('second', '2020-09-08T12:00:12.12345678+00:00')
 ----
 12
 
 query I
-SELECT date_part('millisecond', '2020-09-08T12:00:12.12345678+00:00')
+SELECT date_part('second', timestamp '2020-09-08T12:00:12.12345678+00:00')
```

Comment:
are these test changes still needed? 

From my perspective as long as this PR doesn't change the ability to use a string as an argument to a function that requires a timestamp, it is a good change to me

Reply:
I haven't merge the code from the main